### PR TITLE
Document running it tests from intellij IDE

### PR DESCRIPTION
**Pull request by TSFenwick**
Document how to run integration tests from IntelliJ IDE

### Description

Document how to run Integration Tests from IntelliJ IDE

<hr>

##### Key changed/added classes in this PR
 * `integration-tests/README.md`

<hr>

This PR has:
- [x] been self-reviewed.
   - [ ] using the [concurrency checklist](https://github.com/apache/druid/blob/master/dev/code-review/concurrency.md) (Remove this item if the PR doesn't have any relation to concurrency.)
- [ ] added documentation for new or modified features or behaviors.
- [ ] added Javadocs for most classes and all non-trivial methods. Linked related entities via Javadoc links.
- [ ] added or updated version, license, or notice information in [licenses.yaml](https://github.com/apache/druid/blob/master/dev/license.md)
- [ ] added comments explaining the "why" and the intent of the code wherever would not be obvious for an unfamiliar reader.
- [ ] added unit tests or modified existing tests to cover new code paths, ensuring the threshold for [code coverage](https://github.com/apache/druid/blob/master/dev/code-review/code-coverage.md) is met.
- [ ] added integration tests.
- [ ] been tested in a test Druid cluster.

### CODE REVIEW

File: integration-tests/README.md [hunk 1]
```diff
@@ -123,6 +123,27 @@ Druid routers for security group integration test (permissive tls, no client aut
    mvn verify -P integration-tests -pl integration-tests -Ddocker.build.skip=true -Ddocker.run.skip=true -Dit.test=<test_name>
    ```
 
+### Running tests from IntelliJ
+
+A prerequisite to run tests from intellij you need a cluster up and running. Modify the run configuration's `VM options`
+to the values below.
```
Comment (vtlim) [inline integration-tests/README.md:129]:
```suggestion
Before running tests from IntelliJ, ensure you have a Druid cluster running.
In IntelliJ, edit configurations to add the following VM options:
```

Comment (TSFenwick) [reply integration-tests/README.md:129]: Reads a lot better. Will just make slight naming changes to make sure it matches exactly with intellij's naming

File: integration-tests/README.md [hunk 2]
```diff
@@ -123,6 +123,27 @@ Druid routers for security group integration test (permissive tls, no client aut
...
+-Ddruid.test.config.dockerIp=localhost
+-Ddruid.zk.service.host=localhost
+-Ddruid.client.https.trustStorePath=client_tls/truststore.jks
+-Ddruid.client.https.trustStorePassword=druid123
+-Ddruid.client.https.keyStorePath=client_tls/client.jks
+-Ddruid.client.https.certAlias=druid
+-Ddruid.client.https.keyManagerPassword=druid123
+-Ddruid.client.https.keyStorePassword=druid123
+```
+and to run the test you need to run the test from the test configuration often times found around the top right corner
```
Comment (vtlim) [inline integration-tests/README.md:143]:
```suggestion
Run tests from the test configuration found in the top right corner of the IDE.
```
Is there a certain icon or menu item to look for?

**Comment by FrankChen021** [comment]
I was looking for such guidance for a long time when I first started my work in Druid. 
Thanks for your great work @TSFenwick

**Review by FrankChen021 — APPROVED** [review]

**Comment by FrankChen021** [comment]
I merged this because the failed CI has nothing to do with the doc change in this PR.

**Pull request merged**